jetson/face_extractor.py

```diff
@@ -1,56 +1,31 @@
 import argparse
 from glob import glob
-import hashlib
 import os
 import warnings
 
 import cv2
 import numpy as np
 import torch
-from PIL import Image
 from tqdm import tqdm
 
-from face_detector import FaceDetector
+from face_detector_threaded import FaceDetector
 
 warnings.filterwarnings('once')
 
 
-# for videos, saves filenames and hashes
-# for images, saves filenames
-def get_files(input_dir, hash_file=None, images=False):
-    if images:
-        img_ext = ['.jpg', '.JPG', '.png', '.PNG']
-        imgs = [glob(f"{input_dir}/*{e}", recursive=args.recursive) for e in img_ext]
-        images = [img for subimg in imgs for img in subimg]
-        return images
-    else:
-        video_ext = ['.mp4', '.mov', '.MOV', '.MP4']
-        videos = [glob(f"{input_dir}/*{e}", recursive=args.recursive) for e in video_ext]
-        # Reduce the 2d list from before to a 1d list
-        videos = [vid for subvid in videos for vid in subvid]
-
-        hashes = []
-        for video in videos:
-            hasher = hashlib.md5()
-            buf_size = 65536
-            with open(video, 'rb') as video_file:
-                buf = video_file.read(buf_size)
-                while len(buf) > 0:
-                    hasher.update(buf)
-                    buf = video_file.read(buf_size)
-            hashes.append(hasher.hexdigest())
-
-        if hash_file is None:
-            return list(zip(videos, hashes))
-        else:
-            with open(hash_file, 'r') as f:
-                processed_hashes = [line.strip() for line in f]
-            # Filter out the videos that have already been processed
-            return [(file, file_hash) for (file, file_hash) in zip(videos, hashes) if file_hash not in processed_hashes]
+def get_files(input_dir, images):
+    """Get filenames from input directory."""
+    file_ext = ['.jpg', '.JPG', '.png', '.PNG'] if images else ['.mp4', '.MP4', '.mov', '.MOV']
```

Comment:
can we abstract these extensions into global constants `IMAGE_EXT` and `VIDEO_EXT`. If I had to guess they'll probably be used again and if we change them we would only have to change 1 line of code instead of potentially several. (maintainability)